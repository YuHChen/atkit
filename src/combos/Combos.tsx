import React from "react";

import Wip from "../Wip";

const Combos = () => <Wip className="mt-3" pageName="combos page" />;

export default Combos;
