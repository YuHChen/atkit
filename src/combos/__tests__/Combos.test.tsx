import React from "react";
import { render } from "@testing-library/react";

import Combos from "../Combos";

describe("Combos", () => {
  test("renders as expected", () => {
    const { container } = render(<Combos />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
