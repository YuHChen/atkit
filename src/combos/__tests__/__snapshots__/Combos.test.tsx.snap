// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Combos > renders as expected 1`] = `
<div
  class="fade alert alert-info show"
  role="alert"
>
  <div
    class="alert-heading h4"
  >
    Pardon the mess
  </div>
  <p
    class="mb-0"
  >
    This 
    combos page
     is still a work in progress.
  </p>
</div>
`;
